scripts/cmake-presets/ci-ubuntu-github.json

```diff
@@ -52,7 +52,7 @@
       "cacheVariables": {
         "CELERITAS_USE_Geant4":  {"type": "BOOL", "value": "ON"},
         "CELERITAS_USE_HepMC3":  {"type": "BOOL", "value": "ON"},
-        "CELERITAS_USE_ROOT":    {"type": "BOOL", "value": "ON"},
+        "CELERITAS_USE_ROOT":    null,
```

Comment:
Does this means 'use the default value'? (i.e. do not set it?)

Reply:
Correct; it forces the variable to be "unset". This allows us to use some configurations with ROOT and some without, needed because I didn't want to do two separate ROOT builds (one c++17, one c++20).